### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by beetlejays**

### CODE REVIEW

File: js/answer.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export function showAnswer() {
     const answer = card.querySelector('[data-js="answer-container"]');
     showButton.addEventListener('click', () => {
       answer.classList.toggle('answer-display');
+      showButton.classList.toggle('btn--active');
```
Comment (sarina-nikolic) [inline js/answer.js:11]: btn = button?

File: css/components/card.css [hunk 2]
```diff
@@ -30,26 +32,10 @@
...
 
-.card__bookmark__button:hover,
-.card__bookmark__button:active,
-.card__bookmark__button:focus,
-.card__bookmark__button:focus-visible {
-  filter: hue-rotate(90deg) brightness(2);
-}
-
 .card__bookmark__button--active {
   filter: hue-rotate(90deg) brightness(2);
```
Comment (sarina-nikolic) [inline css/components/card.css:36]: what's this kind of magic? :D

Comment (beetlejays) [reply css/components/card.css:36]: :)  Focus shows the "focus-frame" when you use the Tab-Key on your keyboard - to make the site better accessible for people with disabilities

File: js/answer.js (hunk 1 above)
Comment (beetlejays) [reply js/answer.js:11]: yes, used for button. Will be rewritten soon to make it more concise :)

**Pull request merged**